python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,44 @@
+from ray._private.structured_logging.formatters import LogfmtFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+
+# A dictionary to map encoding types to their corresponding logging configurations.
+LOG_MODE_DICT = {
+    "TEXT": {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "logfmt": {
+                "()": LogfmtFormatter,
+            },
+        },
+        "filters": {
+            "core_context": {
+                "()": CoreContextFilter,
+            },
+        },
+        "handlers": {
+            "console": {
+                "level": "INFO",
```

Comment:
The default log level is Ray Serve is `INFO`.

https://sourcegraph.com/github.com/ray-project/ray@a1590725a8cc8a47734ee8676da4e6a8fbb5548a/-/blob/python/ray/serve/schema.py?L125